ext/include/opentelemetry/ext/zpages/tracez_data_aggregator.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
+
+// include libraries
+#include <string>
+#include <map>
+#include <unordered_set>
+#include <vector>
+#include <list>
+
+// include files
+#include "opentelemetry/ext/zpages/tracez_processor.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+#include "opentelemetry/ext/zpages/latency_boundaries.h"
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/trace/canonical_code.h"
+
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace ext
+{
+namespace zpages
+{
+
+/**
+ * kMaxNumberOfSampleSpans is the maximum number of error and latency sample spans stored at any given time.
+ */
+const int kMaxNumberOfSampleSpans = 5;
+
+/**
+ * AggregatedInformation is the aggregated span information that is stored for each span name. 
+ */
+struct AggregatedInformation{
+  
+  int running_spans_; 
```

Comment:
Struct's variables don't need to have an underscore suffix. Meaning running_spans instead of running_spans_.
See: https://google.github.io/styleguide/cppguide.html#Variable_Names